website/pages/docs/release-notes/1.6.0.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: 1.6.0
+sidebar_title: 1.6.0
+description: |-
+  This page contains release notes for Vault 1.6.0.
+---
+
+# Vault 1.6.0
+
+## Vault 1.6 Release Highlights
+
+**Transform: Tokenization Tech Preview (Enterprise ADP Module Only)**: Vault 1.6 introduces a new transformation method
+for tokenizing sensitive data stored in un-trusted/semi-trusted systems. Tokenization is available as part of the
+“Advanced Data Protection” module in Vault Enterprise.  Tokenization provides non-reversible data protection pursuant to
+requirements for data irreversibility (PCI-DSS, GDPR, etc.). This feature is being released in a Tech Preview.
+
+**Integrated Storage Enhancements**:  Continuing with the enhancements made to Vault’s Integrated storage, we are adding
+the following new features:
+* **Cloud auto-join** provides support for Vault nodes to automatically discover and join a cluster via specified cloud
+metadata. This is particularly useful when IP addresses are not static.
+* **Automated snapshots** provides in-built Vault functionality for automated snapshots that takes snapshots of the
+state of the Vault servers and saves them locally, or pushes them to an optional remote storage service.
+
+**UI improvements**
+* **New UI for Transform secrets engine (Enterprise ADP Module Only)**
+  * To configure FPE and Masking transformations, including custom alphabets and patterns for FPE.
+  * To create and manage roles and patterns for templated use in current and future FPE and masking transformations.
+* **Improvements to Vault Usage UI** to show new metrics for “active clients”, “unique entities” and “active direct
+tokens” that help with understanding Vault usage
+
+**New Command “vault usage”**: Support for analyzing active usage of applications and users with new metric client
```

Comment:
This should likely be called `New “Vault Usage”` and maybe mention it is UI based. There is not CLI option as this time (only API + UI).

Reply:
Ok I’ll remove it.